### ISSUE TITLE
Plan Route - Route params for profile

### ISSUE THREAD

**Issue by vshcherb** [osmandapp/OsmAnd-iOS#1323]
1. #1323
2. #3527
3. #5668

# Main layout

## Top toolbar
<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/60056c55-430a-4bc5-bae4-e4de6b37f935"  width="300" height="auto"/> </td>
  </tr>
</table>   

**Buttons:**
- [x] Use the same button style we use for Map buttons
- [x] Size – the same as default map buttons – 48 pt

**Title:**
- [x] `New route` – for new routes
- [x] File name for edited tracks

### Actions
- [x] Leading action – **Close**
    - [x] Icon – `ic_custom_cancel`
    - [x] Show confirmation alert if there were changes
- [x] Trailing actions
    - [x] **Options** – show options menu on tap, icon – `ic_custom_overflow_menu_stroke`
    - [x] **Save** – Save current track (visible depending on mode)

### Options Menu, [Figma](https://www.figma.com/design/DyC3BkWuaDb5pFvLvYnr8k/Plan-a-route?node-id=1801-58019&m=dev)

<table style="width:100%">
  <tr>
    <th>New route</th>
    <th>Edited track</th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/13f3f164-1fb0-481f-86de-e5dbf09ca77e"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/2cb49532-4d6a-43a7-9565-43d2bf493364"  width="300" height="auto"/> </td>
  </tr>
</table>   

This is a new version of the current Options menu that appears after tapping on the **Options** button in the bottom toolbar. In this revision, we remove duplicated actions present in other places and move the menu button to the top toolbar.

- [x] Use native system context menu (`UIMenu`) for the updated version of the Options menu

**Actions:**
- [x] **Save as…** (previously *Save changes*)
    - [x] Icon – `ic_custom_save_to_file`
    - [x] Opens a dialogue view to input a new file name.
    - [x] Visible in both New route and Edit track modes.
- [x] **Save as copy** (previously *Save as new track*)
   - [x] Icon – `ic_custom_save_as_new_file`
   - [x] Saves a duplicate of the track.
    - [x] Show only in Edit mode, don't show in the New route mode.
- [x] **Append to existing track** (previously *Add to a track*)
    - [x] Icon – `ic_custom_add_to_track`.
    - [x] Show track selection view on tap.
- [x] **Change segment order**
    - [x] Icon – `ic_custom_list`
    - [x] Show UI for changing the segment order list.
- [x] **View directions**
    - [x] Icon – ``
    - [x] Show
- [x] **Reverse route**
    - [x] Icon – ` `
    - [x] Show 
- [x] **Navigation**
    - [x] Icon – `ic_custom_navigation_outlined`
    - [x] Close Plan Route mode and pass the calculated track directly to active Navigation mode.
- [x] **Clear all points**
    - [x] Icon – `ic_custom_trash_outlined`
    - [x] Must be styled as a destructive element (red color text/icon for iOS native menu).

## Bottom toolbar
Previous version has Options button, undo, redo and Add point button. We need to change it style to use Liquid glass button backgrounds for iOS and Map style buttons for iPad layout.

<table style="width:100%">
  <tr>
    <th>iOS</th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/3db82c77-0215-46fb-b30a-4690fc26cc81"  width="300" height="auto"/> </td>
    <td><img src=""  width="300" height="auto"/> </td>
  </tr>
</table>   

**Background:**
- [x] Remove background in the closed state

**Actions:**
- [x] **+ POI** – Opens the "Add Waypoint / POI" UI at the current map center coordinates.
    - [x] Icon – `ic_custom_add`
- [x] **Undo** – Reverts the last action. Works the same way as the previous version.
    - [x] Icon – `ic_custom_undo`
- [x] **Redo** – Replays the last reverted action. Works the same way as the previous version.
    - [x] Icon – `ic_custom_redo`
- [x] **Route +** (previously *Add point*) – Adds a new route point directly to the current map center.
    - [x] Icon – `ic_custom_add`

**Button Style & Hierarchy:**
- [x] **Button height:**
    - [x] iOS – 44 pt
    - [x] iPad – 48 pt
- [x] **Background style:** – Follow the standard Map button style
- [x] Text color – `textColorPrimary`
- [x] Icon color – `iconColorBlack`

## Bottom sheet

<table style="width:100%">
  <tr>
    <th>Closed</th>
    <th>Middle</th>
    <th>Open</th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/a20d47aa-b4e1-4bf0-9c4e-ddd540f3d9d4"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/53a4e33a-aadf-44d7-a891-431c5e6e5a77"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/7548d0dd-37f7-420d-b951-580e4a2e05d9"  width="200" height="auto"/> </td>
  </tr>
</table>  

- [x] Use system native – bottom sheet with ability to tap the map below it.
    - [x] https://developer.apple.com/documentation/swiftui/view/presentationdetents(_:)
    - [x] https://developer.apple.com/documentation/swiftui/view/presentationbackgroundinteraction(_:)
- [x]  Should have 3 states: Bottom, Screen Center, Full screen.

### Top part

<table style="width:100%">
  <tr>
    <th></th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/ade1b31c-e9ed-41a0-b45b-d941a42ea813"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/cd9212ec-5f0b-4f9d-b3ea-6ea17b871ca1"  width="300" height="auto"/> </td>
  </tr>
</table>   

Consist of route info.

- [x] Show top sheet grabber (handle bar).
- [x] Show route status icon – `ic_custom_plan_route` using `iconColorActive`.

**Behavior & Gestures:**
- [x] User must have the ability to dynamically change the bottom sheet height by dragging the sheet via the top grabber (pan gesture).
- [x] Tapping anywhere on the **Top part** must instantly toggle the bottom sheet between the **Collapsed** (minimized, showing only the Top part) and **Middle** (expanded to show the list/tabs) states.
    - [x] Ensure that tap-to-toggle animations are smooth and consistent with native iOS sheet presentation transitions.

**First line:**

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/8c5bc7c2-52d9-4a38-9024-3df1fe833a4d"  width="auto" height="auto"/> </td>
  </tr>
</table>   

- [x] **Styling:**
    - [x] Text size – dynamic type `body`
    - [x] Distance value color – `textColorPrimary`
    - [x] Units/Time color – `textColorSecondary`
- [x] Show total route distance (e.g., `1 209 km`).
- [x] Show **Travel duration** (e.g., `1 h 17 m`).
- [x] Show **Estimated Time of Arrival (ETA)** in parentheses (e.g., `(17:47:00)`).
- [x] Divide the distance block and the time block with a bullet point symbol ` • `.
- [x] Don't show time for the new route and it route between points is set to Straight line

**Second line:**

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/2523802f-a125-48e9-915b-74f89fdeafbc"  width="auto" height="auto"/> </td>
  </tr>
</table>   

- [x] **Styling:**
    - [x] Text size – dynamic type `subhead`
    - [x] Text color – `textColorSecondary`
- [x] Show uphill, put SF Symbol `arrow.up.right` before value 
- [x] Show downhill, put SF Symbol `arrow.down.right` before value
- [x] Show vertical divider ` | ` between elevation data and map center info.
- [x] Show distance from the current map center (maintain current second-line logic).
- [x] Show bearing angle (e.g., `100°`), maintaining current second-line logic.

### Tabs
Current version doesn't have tabs, in this version we will add 3 tabs for POI, Analyze and Route.

- [x] Use segmented control for tabs
- [x] Tap on tab should switch to that tab and open bottom sheet to the Middle state

**Tabs:**
- [x] **POI** –
- [x] **Analyze** – #3527
- [x] **Route** –

**Comment by dmpr0** [comment]
# POI tab, [Figma](https://www.figma.com/design/DyC3BkWuaDb5pFvLvYnr8k/Plan-a-route?node-id=1816-94862&m=dev)

<table style="width:100%">
  <tr>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/f0fdaefc-fa9a-45d4-957d-c150e8f6df99"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/80b0e87f-3054-4fcf-90c9-a76d55b83797"  width="200" height="auto"/> </td>
  </tr>
</table>   



This new functionality offers a complete route planning experience by allowing you to plan not only the route but also add points of interest simultaneously.


## Empty state
<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/bfba392e-5167-485c-a1ee-192fe14ee231"  width="300" height="auto"/> </td>
  </tr>
</table>   

- [x] Title – `Add Points`
- [x] Description – `Select places on the map or copy them from your Favorites to see them along your route.`
- [x] Icon – `ic_custom_folder_open`
- [ ] Action – **Copy from**
- [ ] Show in separate group

## Points


<table style="width:100%">
  <tr>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/4ccf1c6b-55ba-4ea3-bd8e-c7c0874de474"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/a7993670-e1ef-4206-9868-5b72734728af"  width="200" height="auto"/> </td>
  </tr>
</table>   


### Header

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/a60bce4a-4997-4a4d-8a09-f0d0c2607f7d"  width="300" height="auto"/> </td>
  </tr>
</table>   

- [x] Title – Group name
- [x] Subtitle – quantity of points, for example `32 points`

**Action button:**
- [x] Show action button on the right side, use SF Symbol `ellipsis`
- [x]  Show group menu on tap

### Content

- [x] Show list of waypoints
- [x] Use item UI from Favorite points
- [x] Tap on point should open context menu

### Group menu, [Figma](https://www.figma.com/design/DyC3BkWuaDb5pFvLvYnr8k/Plan-a-route?node-id=1816-94855&m=dev)

<table style="width:100%">
  <tr>
    <th></th>
    <th></th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/2bc728db-b0e8-44c0-894c-4d5822581cc1"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/622f05c5-3005-4153-abde-3ac1a7881b1e"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/37d16ef6-ffe9-4d5a-91b4-e34dfc0c0d9f"  width="200" height="auto"/> </td>
  </tr>
</table>  

- [x] **Rename**, icon – `ic_custom_edit`
	- [x] Open Rename alert on tap
- [x] **Change appearance**, icon – 
- [x] **Sort**, icon – `ic_custom_swap`
- - [x] Open additional menu on tap
- [ ] **Copy from**, icon – `ic_custom_copy`
- - [ ] Open additional menu on tap
- [x] **Delete**, icon – `ic_custom_trash_outlined`

**Sort menu**
- [x] **Last modified**, icon – `ic_custom_history`
- [x] **Name A – Z**, icon – `ic_custom_sort_name_ascending`
- [x] **Name Z – A**, icon – `ic_custom_sort_name_descending`
- [x] **Newest date first**, icon – `ic_custom_sort_date_newest`
- [x] **Oldest date first**, icon – `ic_custom_sort_date_oldest`

### Point menu

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/35262ad9-bb1e-45f3-822f-1a2fd3928e82"  width="300" height="auto"/> </td>
  </tr>
</table>   

- [x] **Edit**, icon – `ic_custom_edit`
- - [x] Open Edit Waypoint screen
- [x] **Delete**, icon – `ic_custom_trash_outlined`
- - [x] Delete point from the list

## Action

<table style="width:100%">
  <tr>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/a7f41fa3-7013-4a99-bfc6-248cbe0e072b"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/53d40917-f4ea-429f-89d0-136bb52c5a63"  width="200" height="auto"/> </td>
  </tr>
</table>   

- [x] Show **Add new group** button as last item
- [x] Show Add new group alert on tap

**Comment by dmpr0** [comment]
# Route tab, [Figma](https://www.figma.com/design/DyC3BkWuaDb5pFvLvYnr8k/Plan-a-route?node-id=1805-40840&m=dev)

<table style="width:100%">
  <tr>
    <th>Empty state</th>
    <th>Simple</th>
    <th>Complex</th>
    <th>Complex, with sections</th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/2486ddf6-3adc-4b2e-8c44-0b7edc4e5a67"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/ec8b17fe-cc48-48b4-96ac-57c26147adbd"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/32693e3b-c5da-4734-9e06-3657805ac3a1"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/c0c4c383-8c67-4720-9ed4-c03c933246ec"  width="200" height="auto"/> </td>
  </tr>
</table>  

The tab content is dynamic and varies depending on the route's complexity. A single GPX file can contain multiple segments, and OsmAnd allows splitting each segment into distinct sections with different route types.

## Empty state

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/2e5880cb-42a5-496a-a283-e097a63c068b"  width="300" height="auto"/> </td>
  </tr>
</table>   

- [x] Show when there is no route points
- [x] Title – `No points added yet`
- [x] Description – `Tap anywhere on the map or use search to add the first points of your route.`
- [x] Icon – `ic_custom_plan_route`

## Segment

- [x] Add **Start new segment** button as the last item in a separate group
  - [x] Behavior remains the same as current Options menu implementation

### Segment header


- [x] Show action button on the right side, use SF Symbol `ellipsis`
- [x] Show Segment menu on tap

- - - - - -
 

<table style="width:100%">
  <tr>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/0aed545e-1d35-4dfb-bc68-afb4dbfbc3d0"  width="300" height="auto"/> </td>
  </tr>
</table>  

- [x] Title – `Route points`
- [x] Use for Empty state and for the route with one segment

<table style="width:100%">
  <tr>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/097a12d6-59c1-4171-adac-a2ed919c951e"  width="300" height="auto"/> </td>
  </tr>
</table> 

- [x] Title – `Segment 1`, add segment number
- [x] Subtitle – Route type and distance, for example `Cycling • 53 km`
- [x] Use when route has more than one segment, and they have different route types.

<table style="width:100%">
  <tr>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/70d91f62-c65b-42a2-8699-d695d2a0dd14"  width="300" height="auto"/> </td>
  </tr>
</table> 

- [x] Title – `Segment 1`, add segment number
- [x] Use when route has one or more segment with single route type

### Segment menu

<table style="width:100%">
  <tr>
    <th>Single route type</th>
    <th>Multiply route types</th>
    <th>Sort</th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/e4d30d11-60b2-4ff9-8f83-44fdc9e0ea59"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/87e953d3-6afd-4a44-8c03-358bc192d4ec"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/c9ffaa7d-f499-4578-b239-c0890c11d9f2"  width="300" height="auto"/> </td>
  </tr>
</table>  

- [x] **Change route type** – Show when segment has single route type (without sections).
- - [x] Subtitle – route type name
- - [x] Icon = route type
- [x] **Set single route type** – show when segment has few sections with different route types
- - [x] Icon – `ic_custom_navigation_outlined`
- [x] Open **Route between points** on tap for both variants

- - -

- [x] **Sort** – show additional list on tap
- - [x] Icon – `ic_custom_swap`

**Sort actions**
- [x] **Manual** – default option
  - [x] Icon – `ic_custom_swap`
- [x] **Sort door-to-door** – reorders points to minimize total travel distance (similar to Waypoints sorting in Navigation)
  - [x] Icon – `ic_custom_swap`

- - -

- [x] **Save as…** – open Save file view on tap, after saving, get back to editing.
- - [x] Icon – `ic_custom_save_to_file_outlined`
- [x] **Delete segment**
- - [x] Icon – `ic_custom_trash_outlined`


## Point
<table style="width:100%">
  <tr>
    <th>Start</th>
    <th></th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/26dc2626-478a-4697-806f-9429bd3e1f16"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/a1796e4f-8f23-41b4-9356-639df5b0b793"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/9de2c14b-65dd-4fc0-baa9-84a647b52295"  width="300" height="auto"/> </td>
  </tr>
</table>  

Point item functional remains the same.

**Actions & Controls:**
- [x] **Leading action – Remove button:** Use native list properties. On tap: Instantly delete the point and trigger route recalculation. 
- - [x] Do not show the trailing "Delete" confirmation button.
- [x] **Icon:** Show native numeric point index inside a circle (`1`, `2`, `3`).
- [x] **Trailing action – Move handle:** Native iOS three-line reordering handle. Behavior: Change point order in the list on drag (works as it does now).

**Text:**
- [x] **First line:**
- - [x] Intermediate points: Show distance and bearing (e.g., `3.58 km • 100°`).
- - [x] First point: Replace text with `Start`.
- - [x] Last point: Replace distance and bearing with `Destination`.
- - [x] Text size: subhead, text color –  `textColorSecondary`
- [x] **Second line (Point name):** show point number. T
- - [x] Text size: dynamic body, text color – `textColorPrimary`

## Section header

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/44e7a6fd-bb33-4347-ba30-767faed8f47b"  width="300" height="auto"/> </td>
  </tr>
</table>   

- [x] Show when user changes route type for a point.
- [x] Show route type icon on the left
- [x] Label – display route type name
- [x] Show section distance, right-aligned
- [x] Trailing action – Section menu
  - [x] Icon – `ic_custom_overflow_menu_stroke`
  - [x] Open section menu on tap

### Section menu

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/1a9dcd32-3c5c-4d20-ac81-b009e63cfed9"  width="300" height="auto"/> </td>
  </tr>
</table>   

- [x] **Change route type** – Show when segment has single route type (without sections).
- - [x] Subtitle – route type name
- - [x] Icon = route type
- - [x] Open **Route between points** on tap for both variants
- [x] **Sort** – show additional list on tap
- - [x] Icon – `ic_custom_swap`
- [x] **Delete section**
- - [x] Icon – `ic_custom_trash_outlined`

**Comment by dmpr0** [comment]
# Route between points, [Figma](https://www.figma.com/design/DyC3BkWuaDb5pFvLvYnr8k/Plan-a-route?node-id=1818-100038&m=dev)
A new menu version lets you view and change the routing type for all sections and segments.  It will have two variants: simple and complex. 

* Can be open by taping on Route between points button on the map
* Can be opened from **Segment** or **Section menu**

## Simple version

<table style="width:100%">
  <tr>
    <th>Route type</th>
    <th>Setting</th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/688c5fb4-fc39-4e3e-a17b-8407102d31c9"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/ab28cf1b-5026-4341-a824-01e133182b4b"  width="200" height="auto"/> </td>
  </tr>
</table>   

- [x] Use native modal sheets
- [x] Leading action – Close
- [x] Trailing action – Done
- [x] Title – **Route between points**

**Segmented control**
- [x] Add segmented control to switch between **Route type** and **Settings**
- [x] Open on **Route type** by default

### Route type
- [x] Show Straight line as the first option
- [x] Show list of route types and profiles
- [x] Add **Start new segment** as the last item
  - [x] Add footer below with description – `New segment continues with the same route type.`
  - [x] Close bottom sheet, add new segment and allow user to start adding route points

### Settings
- [x] Show list of predefined settings for the selected route type

## Complex version

<table style="width:100%">
  <tr>
    <th></th>
    <th></th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/504fb15c-7cc1-4cc9-b70a-2bf87e201353"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/e0973152-46cd-4a3a-a79e-40da614ba272"  width="200" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/ae2ef587-7859-4a51-a072-7514904c2cbd"  width="200" height="auto"/> </td>
  </tr>
</table>  


Shown when the route contains more than one route type. This can occur when two segments have different route types, or when a single segment has two sections with different route types.

- Can be opened by tapping the Route between points button on the map
- Can be opened from the **Segment** or **Section menu**


**NavBar**
- [x] Leading action – Close
- [x] Title – **Route between points**

**Content**
- [x] Add description – `Select a segment or section to change the Route type and Settings.`
- [x] Show segments and sections from the route

**Segment without sections**
<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/884969dc-5dcb-4955-8781-6abc77e4de35"  width="300" height="auto"/> </td>
  </tr>
</table>   

- [x] Show header with segment number, for example `Segment 2`
- [x] Show item with segment route type
  - [x] Icon – route type icon
  - [x] Name – route type name
  - [x] Show segment distance next to chevron
- [x] Open Route type screen for the segment on tap


**Segment with sections**
<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/f39f6adf-0275-47c8-8941-604456a886e9"  width="300" height="auto"/> </td>
  </tr>
</table>   

- [x] Show header with segment number, for example `Segment 2`
- [x] Show item for each section with route type and section distance
- [x] Add **Change for whole segment** as the last item
  - [x] Show total segment distance
  - [x] Open Route type screen for the segment on tap. Selecting a route type on the next screen will apply to the whole segment

**Actions:**
<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/be3d0e79-241b-4c4b-9381-69e7791487d6"  width="300" height="auto"/> </td>
  </tr>
</table>   

- [x] **Start new segment** – Close bottom sheet, add new segment and allow user to start adding route points
- [x] **Change for whole track** – Open Route type bottom sheet where user can select a route type for the entire track

### Route type
<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/209804d2-21dc-4634-ac2c-efa8ec9d1705"  width="200" height="auto"/> </td>
  </tr>
</table>   

**NavBar:**
- [x] Use native modal sheets
- [x] Leading action – Back
- - [x] Open previous screen on tap
- [x] Trailing action – Done
- [x] Title –  segment name or count, for example `Segment 1`
- [x] Subtitle – `Section route type • distance`, for example `Bicycle • 17 km`

**Segmented control**
- [x] Show the same Segment control as for Simple version
- [x] Add description: `Bicycle section of Segment 1 will be recalculated using the selected profile.`
    - [x] Dynamic values: Replace `Bicycle` with the current section's transport mode and `Segment 1` with the active segment name.

**Comment by dmpr0** [comment]
# UI Review

## Naming 

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/8413d8e6-79e7-4b84-9336-b418d52c8796"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Get back previous naming logic. Taping Done > should use current day + date for track name, currently it's **New route**

## Main menu

- [x] Hive View directions

## NavBar
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/b5ae2e73-618c-49c3-b5e8-816cde02bc09"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] NavBar should be saved as component that can be reused
- [x] Fix gradient in the dark theme
- [x] Buttons should use the same style as map buttons
	- [x] buttons in navbar
	- [x] Route between points	
- [x] **Save** button - use map button style. Shadow is missing

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/c88d7b41-86b1-49a4-983d-7c9f2fc6bcad"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Increase by 6 pt padding between close button and Title

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/7ceb83c7-c7de-4d2d-afdf-3a3fd5ea80ce"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  



- [x] Increase by 14 pt distance between NavBar and Top row buttons 

## Map center

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/fb394796-d354-4293-aa49-6b19f30e180e"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Map center indicator is not visible in the dark mode. Is it an asset?


- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/f4ca8e23-0982-484a-9365-168c79969777"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Added points are located not in the center
- [x] Why two map center icons are visible?

## Top info

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/6249e198-6a94-42f6-933e-c4375d8860bb"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Uphill and downhill is showing zero? It seems appear only after tap on graph on Analyze tab

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/8750fbd8-5114-478d-b44a-1949b5dbc780"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] ETA and time is missing?

- - - - - -
 

- [x] Distance is showing to second map center


- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/d308deb8-746b-47f6-92b1-58962aa4033a"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Add circular progress when route is calculating. For example after adding new point. It should replace icon without any layout jumps

## Bottom buttons

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/588e060e-ed3f-41ad-accf-e871dbf83600"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Left / Right padding for the POI / Route button


- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/31732647-7623-4d81-a97a-dc15d713ee5a"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x]  POI / Route and Undo / Redo buttons is hardly visible in the dark mode

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/1da171d1-ee1a-4937-ab06-1043d7493845"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Apply proper paddings for **+ POI** button

- - - - - -
 

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/9abcf0e1-01b4-4e80-9bd8-e335e331a729"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Apply proper paddings for **+ Route** button


## Segmented control

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/3f6fe18c-3201-4bd8-bc4e-442ea1cb4b62"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Segmented control has default height?

## Empty states
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/d80d837b-c918-473e-a916-c6caf361316d"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Empty state paddings, text sizes, text color should be identical
- [x] One component for all empty states

**Comment by dmpr0** [comment]
# UI Review

# Route tab


<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/78e88fd8-83fd-4fb0-b944-ecd77fe5bc8e"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Different size of remove icon?

- - - - - -
 

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/7851037a-1444-46d9-ac36-22bf7bc8f7bb"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Missing right padding 

- - - - - -

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/0ea0d449-a99f-4755-910b-b44e8363f2ee"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  
 
- [x] Item height = 67 pt, native Tall tabellcell

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/7a7e38ce-9874-44e2-993e-78de397835ed"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Padding between icon and text 16 dp
- [x] Align separator along the text 

- - - - - -

 **Section item**

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/d1c89766-c79c-476e-8c2c-b7af233a8fc2"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Different corner radius on top and bottom part of group?
- [x] Item height - 52 pt
- [x] Set proper padding between profile icon and profile name
- [x] Use the same for separator below profile name

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/6368d7a7-6bdb-468f-a5a6-ddd065d4872c"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Missing padding from the right side between separator below section

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/1143037a-2a8f-45e6-8393-4a63a3f2a02c"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Add full length separator below last point of the section

## Segment menu

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/342f0dcc-5025-42c7-8750-73bfb3a17b6f"  width="300" height="auto"/> </td>
        <td><img src="https://github.com/user-attachments/assets/e8dcfba4-b899-454f-baa5-088d63e39213"  width="300" height="auto"/> </td>
      </tr>
    </table>   
</details>   

- [x] Missing separators in the menu


- - - - - -

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th>How it works now</th>
        <th>Should be shown</th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/ebcd0f46-8c17-4aac-911b-930041304a64"  width="300" height="auto"/> </td>
        <td><img src="https://github.com/user-attachments/assets/77613ccc-b0d5-49fc-a7d4-e3bb65650e92"  width="300" height="auto"/> </td>
      </tr>
    </table>   
</details>   


- [x] Show route type list after taping on Change route type

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/43575808-aaf7-4ff5-a32a-b5d269d1c2e4
"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Section appears in the segment after changing order of segments


## Point menu
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/5e584517-1083-41ae-9f5e-353e4de4ad2d"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Move point action doesn't work
- [x] Add point after / before doesn't work
- [x] Check all actions, if there no task for change how it works, it should works the same

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/53cd7b8c-1e53-4ebf-b955-d0f209d81373"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Missing icon. Straight line is set as route profile

- - - - - -

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/08476cbf-a4b3-48e3-9a7b-1e512466be81"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Different distance 6.68 km in the header of the bottom sheet and 5.38 km in the segment header. It seems segment continue to show straight line distance instead of calculated route

**Comment by dmpr0** [comment]
# UI Review

# Route between points

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/cb28e549-5234-4567-89ec-d0d22890421c"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Use native colors and component for Segmented control

- - - - - -
 

- [x] Remove scroll inside the list, whole bottom sheet should move up on scroll


- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/ada68538-d6a1-48cc-8ab7-e8c11bd44224"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Settings – use icons

- - - - - -
 

- [x] Add progress after changing route type, show new bottom sheet with progress 


### Button

- - - - - -
 

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/21313e47-4aba-4f2c-887d-df8a709bbcd8
"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Why Car icon is shown for edited track. Route between points show straight line
- [x] Question mark should be shown as undefined profile on Route between option button

- - - - - -

**Comment by dmpr0** [comment]
# UI Review

# POI

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/f5125713-9867-4060-8efe-cd33b88ff1b8"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Empty state for empty group 

- - - - - -
 

- [x] Waypoint item in the list should use the same layout is used for Favorites in My Places

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/7dec14ae-6482-4aa7-bb8e-37d79c84f8ef"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Saving track with Waypoints takes ~7000ms. The track already has 99 waypoints but no route.
* * Track from the Travel - [Berlin.gpx.zip](https://github.com/user-attachments/files/30738448/Berlin.gpx.zip)

**Comment by Zirochkabila** [comment]
# Testing 1

> OsmAnd Maps 5436, released 04.08.26

**Severity**: 🟠 Major / **Priority**: 🟠 Normal

* [x] 1. The map center shifts when the context window is minimized - **Regression**, introduced with the new feature.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/a10af9ed-b774-4835-9108-42c3c827bb09"></video>
</details>

* [x] 2. The map is not centered on the point clicked on the graph, unlike when the track has already been saved.

<details><summary>Details</summary>
<table style="width:100%">
  <tr>
    <th>Plan a route - Actual</th>
    <th>Saved track - Expected</th>
  </tr>
  <tr>
    <td><video source src="https://github.com/user-attachments/assets/506c319d-c075-467b-bb22-f4d635bb7d0b" width="300" height="auto"/></td>
    <td><video source src="https://github.com/user-attachments/assets/96b71d65-5374-4428-91e1-6d37bbe32b27" width="300" height="auto"/></td>
  </tr>
</table>
</details>

* [x] 3. In landscape orientation, the window size cannot be changed.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/e33248e4-13e8-4dbf-96a0-125045e505da"></video>
</details>

* [x] 4. In landscape mode, the buttons on the left side of the screen are displayed one above the other, the buttons on the right are too far from the edge of the screen, and the button at the bottom is almost completely hidden.

<details><summary>Details</summary>
<img width="400" alt="Screenshot 2023-01-31 at 11 19 41" src="https://github.com/user-attachments/assets/8ab6cc56-8d30-4299-be30-6478f9e98bfe">
</details>

* [x] 5. When using RTL, the buttons on the map are jumbled.

<details><summary>Details</summary>
<table style="width:100%">
  <tr>
    <th><img src="https://github.com/user-attachments/assets/11119199-51e6-4656-a932-6786c7aceba6" width="300" height="auto"/></th>
    <th><img src="https://github.com/user-attachments/assets/5c36f4a4-a2ec-46f8-a45f-120db3f5a7e1" width="300" height="auto"/></th>
  </tr>
</table>
</details>

* [x] 6. When selecting **Create New Route**, the first waypoint is added outside the visible area of the map.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/edd4a761-904d-4c50-8914-92b61dcb85eb"></video>
</details>

**Severity**: 🟡 Minor / **Priority**: 🟠 Normal

* [x] 1. The icon for the selected profile changes to the selected profile icon only after a point has been added to the map. Until then, the icon for the previously selected profile is displayed.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/7ad60338-d2e8-4b35-b20e-dc71b62d06e4"></video>
</details>


**Severity**: 🔵 Trivial / **Priority**: 🔵 Low

* [x] 1. In the dialog box for saving a track, there is too much space between the buttons.

<details><summary>Details</summary>
<img width="400" alt="Screenshot 2023-01-31 at 11 19 41" src="https://github.com/user-attachments/assets/a9de670a-d050-4076-9e34-a908438bf751">
</details>

**Comment by dmpr0** [comment]
## Landscape

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/07ed0f64-180c-49db-84e9-b69d5c3b1632"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Bottom sheet should respect safe area as Map buttons do

- - - - - -
 

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/64f321af-7563-436e-a608-94caaf983f02"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Unable to open bottom sheet for the full screen up to NavBar both on iPhone Landscape and Mac OS. It seems drag using handle is doesn't work?

- - - - - -
 
- [x] Zoom and Location buttons are not visible in Landscape

- - - - - -
 
- [x] Hide map button when bottom sheet is open to the full screen

**Comment by DmitrySvetlichny** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5620

**Comment by dmpr0** [comment]
## UI Review

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/3111ee1e-ea22-42e5-863c-c53ba991bd65"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] UI theme should depend on Map mode, not iOS theme.

**Comment by Vladyslav-lys** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5627

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5628

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5629

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5633

**Comment by dmpr0** [comment]
## UI Review

- [x] 3D Mode button should be visible if it visible on the main map


- - - - - - 


<details>
  <summary>Screenshot</summary>
  <table style="width:100%">
    <tr>
      <th></th>
    </tr>
    <tr>
      <td><img src="https://github.com/user-attachments/assets/f0c66fba-fcbe-46e9-863e-980231575056" width="300" height="auto"/></td>
    </tr>
  </table>
</details>


- [x] Add gap info between segments. Should show staright line distance and angle between previous segment last point and next segment first point
- [x]  Use icon `ic_custom_segments_gap`
- [x]  Draw 1 pt stroke, use `customSeparatorSolid` color for stroke

- - - - - -

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <td><video src="https://github.com/user-attachments/assets/eaf546ea-94e5-4b98-adcb-1a8ada3c6be9"  width="300" height="auto"/> </td>
        <td><video src="https://github.com/user-attachments/assets/dfb92614-c02b-4164-ae3a-377dd7a1336c"  width="300" height="auto"/> </td>
      </tr>
    </table>   
</details>   

- [x] Don't calculate route and don't show progress for one point


- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/1174edb1-6010-41ee-a7a4-3fa23f365dda"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Theme for bottom buttons **+POI** and **+Route** should be aligned with menu not map buttons for iPhone. On iPad they will be aligned with map buttons

- - - - - -


 
**Top part**

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/51c25abe-040e-47db-9ae8-5434f618e96b"  width="300" height="auto"/> </td>
        <td><img src="https://github.com/user-attachments/assets/3f43966a-6230-445a-a891-f5261f8c6d23"  width="300" height="auto"/> </td>
      </tr>
    </table>   
</details>   

- [x] Use monospacedDigitSystemFont for **Distance**, **Travel duration**, and **ETA**
- [x] Use monospacedDigitSystemFont for Uphill / Downhill
- [x] Use proper formatting for Travel duration (e.g., 1 h 17 m). See how it is done in Route details.
- [x] Drop seconds from Estimated Time of Arrival (ETA) – 17:47

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/fd62755e-af56-47cd-9b26-1a9ce97bbbfc"  width="300" height="auto"/> </td>
        <td><img src="https://github.com/user-attachments/assets/82258a2b-0086-40a5-9c9b-7fb8e01f877f"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Align padding size between segmented control and empty state / graph. Use bigger padding from POI tab

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/231548da-6731-452a-a5ce-0279aebec9c7"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Why text in those headers has different size and paddings

- - - - - - -

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/e7088c04-c814-4455-9328-da60795bac4b"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


 
- [x] Button paddings are ignored? Text should be trimmed 


- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th>TestFlight</th>
        <th>Release</th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/3c7802a0-7b40-4023-854e-2d4ce50aa222"  width="300" height="auto"/> </td>
        <td><video src="https://github.com/user-attachments/assets/46c6e1f6-0072-4f59-8da9-d67ca41a29f1"  width="300" height="auto"/> </td>
      </tr>
    </table>   
</details>   


- [x] Don't move map after opening Plan a Route

- - - - - -
 


<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th>New</th>
        <th>Editing</th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/2fb8f300-a5d9-40ee-a36d-caa99749df53"  width="300" height="auto"/> </td>
        <td><img src="https://github.com/user-attachments/assets/faf0bec9-5996-4892-becd-a4ad05d9c46b"  width="300" height="auto"/> </td>
      </tr>
    </table>   
</details>   

- [x] Show menu in closed stated for New route and in the Middled for editing after opening 

- - - - - -
 

- [x] **Append to existing track** doesn't work or work the other way then in Release (see Add to a track)

- - - - - -
 

- [x] NavBar title color – always white


- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/0cb835fb-cc0c-48e7-bf35-8955ff06adc9"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Show total length for Complex segment on the second line below title

- - - - - -


<details>
  <summary>Screenshot</summary>
  <table style="width:100%">
    <tr>
      <th></th>
    </tr>
    <tr>
      <td><video src="https://github.com/user-attachments/assets/2529bf8e-51f2-404b-b0f6-7e7305253a7b" width="300" height="auto"/></td>
    </tr>
  </table>
</details>

- [x] Show empty segments (without points) in Route between points, so user can change route type for them


- - - - 

- [x] Each segment should have Start and Destination points subtitle on first and last points

### Save button



<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th>Bottom sheet</th>
        <th>Previous Done button behavior</th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/c6fb22b0-f461-4bdb-8ad9-fd406988ed7b"  width="300" height="auto"/> </td>
        <td><video src="https://github.com/user-attachments/assets/2d48daa5-a235-4e3a-af41-5c18346c9b99"  width="300" height="auto"/> </td>
      </tr>
    </table>   
</details>   


- [x] Always skip **Save as new track** dialog  after taping on **Save** button in Navbar.
	- [x] Show bottom sheet after saving instead of alert.

- - - - - -

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/769aee88-011b-4729-9481-4877dc0f5ca2"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  
 
- [x] Taping Save rewrite previous tracks with the same name. See previous review item, use restore previous logic

 
## iOS 18

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/12a53568-0ea2-4907-a2f5-2dbab8fce4e5"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  


- [x] Save button shape

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/user-attachments/assets/ba39da73-6654-457b-bfba-59219ff6d471"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Done button is hardly visible

**Comment by Zirochkabila** [comment]
# Testing 2

> OsmAnd Maps 5443, released 13.08.26

**Severity**: 🟠 Major / **Priority**: 🟠 Normal

* [ ] 1. When the window is resized to landscape orientation, the center point of the map moves downward.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/fbaaf797-821e-4e4c-a47b-388cb3c8d0d4"></video>
</details>

* [x] 2. The map is not centered on a point selected on the elevation graph.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/a15b8c57-e194-4865-a281-f5348e82cb5f"></video>
</details>

* [x] 3. After resizing the height graph, the points on the graph move in the direction of the finger movement with increased intensity.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/b0fa5207-d1aa-4f1c-a115-0801e5dbc33f"></video>
</details>

* [x] 4. When using RTL, the buttons on the map are jumbled, and the zoom-out button overlaps with the mode selection button.

<details><summary>Details</summary>
<table style="width:100%">
  <tr>
    <th><img src="https://github.com/user-attachments/assets/c17eee4f-800c-4411-8a0e-40339506c372" width="300" height="auto"/></th>
    <th><img src="https://github.com/user-attachments/assets/179f9517-df14-4098-ba54-6cf5c7e31da8" width="300" height="auto"/></th>
  </tr>
</table>
</details>

* [x] 5. The map shifts when the bottom sheet is opened in full-screen mode.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/b01ef740-a8c8-44b4-8a4e-40e3d0a5b506"></video>
</details>

* [x] 6. Hide the map button when the bottom sheet is open in full-screen mode. Currently, the buttons are not hidden and are displayed behind the bottom sheet.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/a115ef2c-5b49-4f29-be00-40745428e615"></video>
</details>

**Severity**: 🟡 Minor / **Priority**: 🟠 Normal

* [x] 1. The icon for the selected profile changes to the selected profile icon only after a point has been added to the map. Until then, the icon for the previously selected profile is displayed.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/8c74bbc7-2555-457d-8320-44809237caa5"></video>
</details>

**Severity**: 🔵 Trivial / **Priority**: 🔵 Low

* [x] 1. The button background is too close to the **No points added yet** line.

<details><summary>Details</summary>
<img width="400" alt="Screenshot 2023-01-31 at 11 19 41" src="https://github.com/user-attachments/assets/0b69b541-22cf-4e6b-ac02-b096bad6259a">
</details>

**Comment by DmitryAlexei** [comment]
- save track
- add to a track

**Comment by dmpr0** [comment]
## UI Review
**Point item**
- [x] Should show distance from the start, currently it show distance between points
- [ ] Additionally @dmpr0  Android shows altitude

- - - - - -
 
- [x] Remove `blue_circle_fill` from assets
- [x] Remove `map_ruler_center_day` and `map_ruler_center_night`
- [x] If resources are not present in assets, contact @dmpr0

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5653

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5656

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5658

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5666

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5663

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5675

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5676

**Comment by aleksandr-tata** [comment]
https://github.com/osmandapp/OsmAnd-iOS/pull/5677

**Comment by Zirochkabila** [comment]
# Testing 3

> OsmAnd Maps 5451, released 25.08.26

**Severity**: 🟠 Major / **Priority**: 🟠 Normal

* [ ] 1. The marker on the elevation graph moves in the opposite direction of the swipe after zooming in or out.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/67d1e3fa-b583-4296-af1c-ca437944466f"></video>
</details>

* [ ] 2. The map shifts when the bottom sheet is opened.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/d8d6ebfb-678d-4333-baaf-430e5e7f2b22"></video>
</details>

**Severity**: 🔵 Trivial / **Priority**: 🔵 Low

* [ ] 1. The button background is too close to the **No points added yet** line.

<details><summary>Details</summary>
<table style="width:100%">
  <tr>
    <td><video source src="https://github.com/user-attachments/assets/9164a598-203b-4781-a678-63086fbd4cee" width="300" height="auto"/></td>
    <th><img src="https://github.com/user-attachments/assets/645b7216-0fbe-412d-8532-4d2a4eca441a" width="300" height="auto"/></th>
  </tr>
</table>
</details>
